velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -57,6 +57,79 @@ class DateTimeFunctionsTest : public SparkFunctionBaseTest {
   }
 };
 
+TEST_F(DateTimeFunctionsTest, toUTCTimestamp) {
+  const auto toUTCTimestamp = [&](const std::string_view ts,
+                                  const std::optional<std::string> tz) {
```

Comment:
tz doesn't need to be optional; 

const std::string& tz

Reply:
Updated.